### DISCUSSION TITLE
Change Keybinds in Pause Screen (Options Menu)

### DISCUSSION THREAD

**Discussion by ghost** [Leather128/LeathersFunkinEngine#257] [Ideas]
In the pause menu whilst playing a song, if you go to the options menu there are no options to change your keybinds.
I think it would be a good idea to add a 'Change Keybinds' option to the pause screen's options menu.
Honestly, I wouldn't care if I had to restart a song just to have the keybinds i desire, so yeah.

**Comment by Leather128** [comment]
sounds like a good idea, and i'm 99% sure you wouldn't have to restart the song lol
